From 5d759f673438f49617040734b92056b25d211283 Mon Sep 17 00:00:00 2001
From: Christian Hesse <mail@eworm.de>
Date: Fri, 13 Oct 2017 09:10:26 +0200
Subject: [PATCH] man: update changes about gateway -> _gateway (#7085)

Commit 5248e7e1f11aba6859de0b28f0dd3778b22842f2
(resolved,nss-myhostname: use _gateway for the gateway) changed how the
gateway is resolved. Reflect the change in documentation.

Close #7084.

(cherry picked from commit 1dc92a06e210a978e54c72168aaaa7487bf2fd2a)
---
 man/nss-myhostname.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/man/nss-myhostname.xml b/man/nss-myhostname.xml
index 6e05cb1897..2b180a37cb 100644
--- a/man/nss-myhostname.xml
+++ b/man/nss-myhostname.xml
@@ -75,7 +75,7 @@
       ending in <literal>.localhost</literal> or <literal>.localhost.localdomain</literal>)
       are resolved to the IP addresses 127.0.0.1 and ::1.</para></listitem>
 
-      <listitem><para>The hostname <literal>gateway</literal> is
+      <listitem><para>The hostname <literal>_gateway</literal> is
       resolved to all current default routing gateway addresses,
       ordered by their metric. This assigns a stable hostname to the
       current gateway, useful for referencing it independently of the
